gravatar/src/main/java/com/gravatar/GravatarConstants.kt

```diff
@@ -14,5 +14,5 @@ internal object GravatarConstants {
     const val GRAVATAR_API_BASE_URL = "https://api.gravatar.com/v1/"
 
     /** Gravatar base host */
-    const val GRAVATAR_BASE_URL = "https://gravatar.com"
+    const val GRAVATAR_BASE_URL = "https://api.gravatar.com/v3/"
```

Comment:
We should rename:
- `GRAVATAR_API_BASE_URL` to `GRAVATAR_API_BASE_URL_V1`
- `GRAVATAR_BASE_URL`  to `GRAVATAR_API_BASE_URL_V3` (double check it's only used in ProfileService)

And at the same time, clean up stuff in `GravatarSdkContainer` (rename `getRetrofit*Builder`).

Reply:
Great idea! Done in [4e7f5d6](https://github.com/Automattic/Gravatar-SDK-Android/pull/141/commits/4e7f5d622a0fceea3dad38528f719adb428e5c2f)